src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx

```diff
@@ -460,77 +469,202 @@ class IssueRuleEditor extends AsyncView<Props, State> {
-                    project={project}
-                    disabled={!hasAccess}
-                  />
-                </PanelRuleItem>
-
-                <PanelSubHeader>{t('Perform these actions')}</PanelSubHeader>
-
-                {this.hasError('actions') && (
-                  <PanelAlert type="error">
-                    {this.state.detailedError?.actions[0]}
-                  </PanelAlert>
-                )}
+                <Step>
+                  <StepConnector />
 
-                <PanelRuleItem>
-                  <RuleNodeList
-                    nodes={this.state.configs?.actions ?? null}
-                    items={actions || []}
-                    placeholder={t('Add an action...')}
-                    onPropertyChange={this.handleChangeActionProperty}
-                    onAddRow={this.handleAddAction}
-                    onDeleteRow={this.handleDeleteAction}
-                    organization={organization}
-                    project={project}
-                    disabled={!hasAccess}
-                  />
-                </PanelRuleItem>
+                  <StepContainer>
+                    <ChevronContainer>
+                      <IconChevron
+                        color="gray400"
+                        isCircled
+                        direction="right"
+                        size="sm"
+                      />
+                    </ChevronContainer>
+
+                    <StepContent>
+                      <StepLead>
+                        {tct('[when:When] an issue matches [selector] of the following', {
+                          when: <Badge />,
+                          selector: (
+                            <EmbeddedWrapper>
+                              <EmbeddedSelectField
+                                className={classNames({
+                                  error: this.hasError('actionMatch'),
+                                })}
+                                inline={false}
+                                styles={{
+                                  control: provided => ({
+                                    ...provided,
+                                    minHeight: '20px',
+                                    height: '20px',
+                                  }),
+                                }}
+                                isSearchable={false}
+                                isClearable={false}
+                                name="actionMatch"
+                                required
+                                flexibleControlStateSize
+                                choices={ACTION_MATCH_CHOICES}
+                                onChange={val => this.handleChange('actionMatch', val)}
+                                disabled={!hasAccess}
+                              />
+                            </EmbeddedWrapper>
+                          ),
+                        })}
+                      </StepLead>
+                      <RuleNodeList
```

Comment:
general design nit: I know this was just carried over from the old form, but some input fields seem to be a lot wider than necessary. For example, the value selector here which is wider than the other two:
![image](https://user-images.githubusercontent.com/9372512/91470704-0adec780-e863-11ea-865d-75b1ce7c6db1.png)

maybe we can fix some of the selectors in a follow up PR